src/test/java/com/direwolf20/buildinggadgets/building/coreTests/RegionIteratorTest.java

```diff
@@ -0,0 +1,119 @@
+package com.direwolf20.buildinggadgets.building.coreTests;
```

Comment:
`coreTests` should be `coretests` and logically if this is in a test dir then the word test isn't needed anywhere other than the class file names.

Reply:
Sorry, but now you are contradicting yourself... 
Also I think it should still have test in in the package, to make clear where it belongs to...

Reply:
It's in the test dir. It's essentially going. Test.test.testclass

Reply:
We [above](https://github.com/Direwolf20-MC/BuildingGadgets/pull/240#pullrequestreview-206224496) came to the conclusion, that it should be in the test package, within the Test sourceSet. 
Even though as a dev we can see that it is in the Test dir, that can't be noticed from within code. Putting it in a seperate test package, has the advantage that import's directly show whether the class is a Test class or not. Otherwise it might be hard to tell, what is Source-Code and what is Test-Code, for classes like UniqueBlockState (and putting the other things under tests makes it more consistent in my opinion)

Reply:
The class is literally titled test and the only time your working with test code is when you're writing tests...

Reply:
Personally I think tests should be in the same package as the source and be named `FooTest` in contrast to the source `Foo`. This way we can test some package-private stuff without using reflection. 
However, since I neither own the repository nor a collaborator, please only use this as another way to look at this problem.